### ISSUE TITLE
Vulgarize research paper

### ISSUE THREAD

**Issue by jemsab** [MLanguage/mlang#39]
Write a simplified version of the research paper so as to explain the project to DGFiP agents who may have no training in compilation techniques and theory

**Comment by rmonat** [comment]
@denismerigoux has already written some [blog post](https://blog.merigoux.ovh/fr/2021/03/01/moderniser-dgfip.html) on it. I don't know what you're exactly looking for?

**Comment by jemsab** [comment]
You're right @rmonat, I'm not sure what licence Denis' blog is under but that could be definitely be part of a solution. 

I think what we're aiming for is something that is more operational (if that makes sense?), a sort of guide to the actual compiler code but that is more understandable to the layman than your research paper.

**Comment by rmonat** [comment]
I guess this is already included in the documentation but there are some additional instructions at https://zenodo.org/record/4456774#.YQBBgx6-g0M
